src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/ProfessionalExternalUserFunctionalTest.java

```diff
@@ -524,4 +528,50 @@ public void deletePbaOfExistingOrganisationShouldBeSuccess() {
         assertThat(paymentAccounts).isEmpty();
         log.info("deletePbaOfExistingOrganisationShouldBeSuccess :: END");
     }
+
+    @Test
+    @ToggleEnable(mapKey = "OrganisationExternalController.addPaymentAccountsToOrganisation", withFeature = false)
+    public void addPbaOfExistingOrganisationShouldBeForbiddenWhenLDOff() {
+        log.info("addPbaOfExistingOrganisationShouldBeForbiddenWhenLDOff :: STARTED");
+
+        setUpOrgTestData();
+        setUpUserBearerTokens(List.of(puiFinanceManager));
+
+        PbaAddRequest pbaAddRequest = new PbaAddRequest();
```

Comment:
`PbaRequest` class could be used instead. Looks like `PbaAddRequest` is a duplicate.